purchase_order_generator_revision/static/description/index.html

```diff
@@ -0,0 +1,124 @@
+<section class="oe_container">
+    <div class="oe_row oe_spaced">
+        <div class="oe_span12">
+            <h2 class="oe_slogan">Module name</h2>
+            <p>This module was written to extend the functionality of ... to support ... and allow you to ...</p>
+        </div>
```

Comment:
Update this file with your actual information, if not in odoo.com/apps will be shown incorrectly.